docs/src/.vuepress/components/kanban.vue

```diff
@@ -123,6 +130,13 @@ export default {
     dropNotAllowed({ payload, container }) {
       console.log("drop not allowed", payload);
     },
+    countValue(column, index) {
+      if (!this.hasExternalCount) return column.columnItems.length;
+      const currentValue = this.columnCount.find(
+        (item) => item.id === column.id
+      )?.value;
+      return currentValue || this.columnCount[index];
+    },
```

Comment:
Can it be improved to accept values in numeric or object forms (not at the same time)?
Also, it must be able to receive `0` as a valid input.

Reply:
what about
```suggestion
 countValue(column, index) {
      if (!this.hasExternalCount) return column.columnItems.length;

      const isNumeric = (value) => !isNaN(Number(value));
      const trueValue = (value) => value || value == 0;
      const validNumericArray = this.columnCount.length === this.columns.length && this.columnCount.every((value) => isNumeric(value)) 

      if (validNumericArray)
        return trueValue(this.columnCount[index])
          ? this.columnCount[index]
          : column.columnItems.length;

      let currentValue = this.columnCount.find(
        (item) => item.id === column.id
      )?.value;

      return trueValue(currentValue) ? currentValue : column.columnItems.length;
    },
```

Reply:
Looks good!